### PR TITLE
[S2GRAPH-84] Test-case compilation error on `s2counter_loader` project

### PR DESCRIPTION
**Pull request by hsleep**
Fix compilation error and remove unused test-case. Also, fixed to pass the test

### CODE REVIEW

File: s2counter_loader/src/test/scala/org/apache/s2graph/counter/loader/stream/ExactCounterStreamingSpec.scala [hunk 1]
```diff
@@ -1,217 +0,0 @@
...
-import s2.models.{Counter, DBModel}
-
-import scala.collection.mutable.{HashMap => MutableHashMap}
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.util.{Failure, Success}
-
-/**
-  * Created by hsleep(honeysleep@gmail.com) on 2015. 11. 19..
-  */
-class ExactCounterStreamingSpec extends FlatSpec with Matchers with BeforeAndAfterAll {
```
Comment (SteamShon) [inline s2counter_loader/src/test/scala/org/apache/s2graph/counter/loader/stream/ExactCounterStreamingSpec.scala]: Is it ok to just delete this specs? Just want to make sure.

Comment (hsleep) [inline s2counter_loader/src/test/scala/org/apache/s2graph/counter/loader/stream/ExactCounterStreamingSpec.scala]: These test-cases cannot build successfully in current version.
I think that we need to rewrite TC and make process to test the spark related sub-project like loader and s2counter_loader.

**Comment by SteamShon** [comment]
@hsleep is this ok to be shipped into version 0.1.0 release? if so, then I will merge this PR and we can discuss rewrite test cases for `s2counter_loader` subproject on separate issue.

**Comment by hsleep** [comment]
Yes. current test-case do not work correctly. We need to rewrite related test-case on other issue.

**Comment by SteamShon** [comment]
@hsleep I see. Please update CHANGES so I can merge this.

**Pull request closed without merging**